notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>python_random_split won't provide that guarantee, you can use python_stratified_split, but even then you can either guarantee that every user is in train and test (filter_by='user'), or that every item is in train and test (filter_by='item'). In most cases it works out that by splitting by user most items appear in both sets, but you may need to check that case.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/729/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I agree. We just followed in the example in the notebook ncf_movielens.ipynb. It should be noted that our algorithm will not given an error if instances of some item/users are missing from the training set. It will still give some result during the test time for that item/user. This is due to the regularization involved in the optimization formulation. </p>